tests/lib/prepare.sh

```diff
@@ -518,3 +529,266 @@ prepare_all_snap() {
 
     disable_kernel_rate_limiting
 }
+
+setup_reflash_magic_core18() {
+        # install the stuff we need
+        distro_install_package kpartx busybox-static
+        distro_install_local_package "$GOHOME"/snapd_*.deb
+        distro_clean_package_cache
+
+        snap install "--${CORE_CHANNEL}" core18
+        snap install "--${CORE_CHANNEL}" snapd
+        # install ubuntu-image
+        snap install --classic --edge ubuntu-image
```

Comment:
Is it possible to use non-edge ubuntu-image? Is the ubuntu-image snap maintained? Last time I looked it was broken and not refreshed periodically.

Reply:
It is possible to use different channels. Its mostly historic that we use edge. The foundations team maintains ubuntu-image.

Reply:
Right but is it actually maintained as a snap?
```
$ snap info ubuntu-image
name:      ubuntu-image
summary:   Create Ubuntu images
publisher: canonical
contact:   snappy-canonical-storeaccount@canonical.com
license:   GPL-3.0
description: |
  Use this tool to create Ubuntu images.
snap-id: 4RW78vIax8JW5S8HkYsa8lNbv68uPaYX
channels:                        
  stable:    –                   
  candidate: –                   
  beta:      1.3+snap1 (95) 35MB classic
  edge:      1.3+snap1 (98) 34MB classic
```